### ISSUE TITLE
How to set or Fix Max File Size Output ?? OPTION PLS

### ISSUE THREAD

**Issue by combodevs** [cdgriffith/FastFlix#234]
How to set or Fix Max File Size Output ?? OPTION PLS

**Comment by forart** [comment]
Yep, it would be great to have a "bitrate based on target file size" like [ShanaEncoder](https://sourceforge.net/projects/shanaencoder/):
![Shana](https://user-images.githubusercontent.com/171307/121867331-23d83c80-cd00-11eb-954c-a69ee03a5e15.png)

**Comment by cdgriffith** [comment]
It's hard to gauge exact total file size, as that will include the video bitrate (which depending on encoder may not be exactly what is requested), all the audio tracks + subtitle tracks + cover + container overhead. 

A good first stop might just be show estimated file size (for video alone) beside whatever bitrate is selected?
